workbench-core/accounts/src/utilities/hostingAccountLifecycleService.ts

```diff
@@ -47,7 +54,17 @@ export default class HostingAccountLifecycleService {
     this._stackName = process.env.STACK_NAME!;
     const ddbTableName = process.env.STACK_NAME!; // The DDB table has the same name as the stackName
     this._aws = new AwsService({ region: process.env.AWS_REGION!, ddbTableName });
-    this._accountService = new AccountService(ddbTableName);
+    this._accountService = new AccountService(this._aws.helpers.ddb);
+  }
+
+  public async listAccounts(listAccountRequest: ListAccountRequest): Promise<PaginatedResponse<Account>> {
+    const queryParams = addPaginationToken(listAccountRequest.paginationToken, {
+      index: 'getResourceByName',
+      key: { name: 'resourceType', value: 'account' },
+      limit: listAccountRequest.pageSize || DEFAULT_API_PAGE_SIZE
+    });
+
+    return await this._accountService.getPaginatedAccounts(queryParams);
```

Comment:
I don't think you need to `return await ....`, you can just return the promise as `return ...`